app/services/ClaimToAdjustService.scala

```diff
@@ -173,8 +171,6 @@ class ClaimToAdjustService @Inject()(val financialDetailsConnector: FinancialDet
 
   private val LAST_DAY_OF_JANUARY: Int = 31
 
-  private val taxReturnDeadlineOf: Option[LocalDate] => Option[LocalDate] = dueDate => {
-    dueDate.map(d => LocalDate.of(d.getYear, Month.JANUARY, LAST_DAY_OF_JANUARY).plusYears(1))
-  }
-
+  private val taxReturnDeadlineOf: Option[LocalDate] => Option[LocalDate] =
```

Comment:
I think this is relevant to the current moment in time, so maybe its worth to move this under DateServiceInterface ?